docs/reference/field_types.rst

```diff
@@ -65,7 +65,7 @@ This is currently limited to scalar types (text, integer, url...) and choice typ
 ``TemplateRegistry::TYPE_ARRAY``
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-You can use the following parameters:
+You can use the following options:
 
 ======================================  ============================================================
 Parameter                               Description
```

Comment:
I think, if possible, we should replace all the "parameter" occurrences with "option".
```suggestion
Option                               Description
```

Reply:
It should be better now